modules/cognitive-services/account/README.md

```diff
@@ -247,6 +247,65 @@ userAssignedIdentities: {
 </details>
 <p>
 
+### Parameter Usage: `roleAssignments`
```

Comment:
Seems to be this was added again in the wrong location. Please move to notes (although we'll remove it eventually anyways)